TreeOfLife.dot

```diff
@@ -0,0 +1,70 @@
+digraph G {
```

Comment:
Wie ist der Baum zu verstehen? Kann man immer nur in einen Zweig weiterentwickeln?

Reply:
Wäre aber ein erster Start und somit genehmigt... ;)